dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/log/LogClient.java

```diff
@@ -179,27 +178,14 @@ public byte[] getLogBytes(String host, int port, String path) {
      * @param path path
      * @return remove task status
      */
-    public Boolean removeTaskLog(@NonNull Host host, String path) {
```

Comment:
Let's also remove the `@return` statement in javadoc

Reply:
Done